server/src/domain/game.rs

```diff
@@ -4,10 +4,18 @@ use super::player::Player;
 // Types //
 ////////////////////////////////////////////////////////////////
 
-pub struct Game {
+pub enum Game {
+    Lobby(LobbyModel),
+}
```

Comment:
A game doesnt start "started", it starts off as a temporary lobby where its being created.